tests/test_aliases.py

```diff
@@ -44,7 +45,8 @@ class Model(BaseModel):
     assert Model().a == 'foobar'
     assert Model(_a='different').a == 'different'
     assert repr(Model.model_fields['a']) == (
-        "FieldInfo(annotation=str, required=False, default='foobar', alias='_a', alias_priority=2)"
+        "FieldInfo(annotation=str, required=False, default='foobar', alias='_a', "
```

Comment:
i think we should only include `validation_alias` and `serialization_alias` in repr if they're different to `alias`.

Reply:
done